packages/desktop-client/public/data/migrations/1679014116737_accounts_iban.sql

```diff
@@ -0,0 +1,7 @@
+BEGIN TRANSACTION;
+
+ALTER TABLE accounts ADD COLUMN iban TEXT;
```

Comment:
maybe we can use the `mask` column to keep there the IBAN?

Reply:
This would absolutely result in too many false positives. I, myself, have 3 different accounts (two at the same bank, one at a different one) ending with `0001`. My mom *also* has an account ending with `0001`.